packages/orbit-components/src/Badge/types.d.ts

```diff
@@ -24,7 +24,6 @@ export type Type =
 export interface Props extends Common.Globals {
   readonly children?: React.ReactNode;
   readonly type?: Type;
-  readonly border?: boolean;
```

Comment:
Hm, shouldn't this be a breaking change?

I found it's still being used in some places, e.g. in Booking [here](https://gitlab.skypicker.com/frontend/booking/-/blob/HEAD/src/ancillaries/flight/features/Seating/components/IncludedSeatingBadge/IncludedSeatingBadge.tsx#L17) and [here](https://gitlab.skypicker.com/frontend/booking/-/blob/HEAD/src/common/ServerDrivenItinerary/ServerDrivenItinerary.tsx#L121).

Reply:
Damn, also [this](https://gitlab.skypicker.com/frontend/home/-/blob/3c28b02d28321c2df57400a14c14d253ded40691/src/search/components/PrebookingModal/components/PrebookingModalItinerary/components/ItinerarySegmentDetail/components/SeatInfoRenderer/SeatInfoRenderer.tsx).

For some reason, I thought this was already announced and it was just a leftover.

Not sure if it makes sense to release it like this. I might create a separate PR for this, WDYT?

Reply:
https://github.com/kiwicom/orbit/pull/4626

I targeted the `aria-label-bcs` branch as this change is not urgent and can be shipped with the other BCs on v20